### DISCUSSION TITLE
The episodes and movies I've seen are hidden

### DISCUSSION THREAD

**Discussion by thewalkerz** [CastagnaIT/plugin.video.netflix#1110] [General]
Hello, I have installed the addon on my fire tv with kodi, but when I log in with my account and try to search for a series or movie that I have already seen, it does not appear, they appear hidden. For example, I was watching the series "Altered Carbon" but when I try to find it in the addon, it doesn't appear, it doesn't even appear in the "continue watching" session, Everything is blank. But everything else works perfectly.

**Comment by haggaie** [comment]
I believe I've also encountered this issue. Shows that I've watched in the past using this plugin suddenly neither appear in the menus nor in search.

**Reply by jakermx** [reply]
Ok , wile in Kodi click left arrow, and then change the display filter to All

**Reply by haggaie** [reply]
Thanks, I can't believe I missed that 😬

**Reply by thewalkerz** [reply]
Wow, that solve my issue, I never thought about that 😅
